### PR TITLE
Support `Round` and `BitShift` operator

### PR DESCRIPTION
**Pull request by take-cheeze**
Larger unsigned types are not supported in ChainerX so they are not testable.

### CODE REVIEW

**Comment by take-cheeze** [comment]
ChainerCI, test this please!

File: runtime/ops/quantize.cc [hunk 1]
```diff
@@ -134,5 +134,37 @@ chainerx::Array ConvIntegerOp::RunImpl(
...
+            result_data[i] = x_ptr[i] << y_ptr[i];
+        }
+    } else {
+        CHECK_EQ("RIGHT", direction);
+        for (size_t i = 0; i < result_data.size(); ++i) {
+            result_data[i] = x_ptr[i] >> y_ptr[i];
+        }
+    }
+
+    return MakeArray(chainerx::Dtype::kInt64, x.shape(), result_data.data()).AsType(x.dtype());
```
Comment (shinh) [inline runtime/ops/quantize.cc]: Again, we should cast back to the original dtype?

File: runtime/ops/quantize.cc [hunk 2]
```diff
@@ -134,5 +134,37 @@ chainerx::Array ConvIntegerOp::RunImpl(
...
+    chainerx::Array double_x = x.AsType(chainerx::Dtype::kFloat64);
+    const double* x_ptr = reinterpret_cast<const double*>(double_x.raw_data());
+    for (size_t i = 0; i < result_data.size(); ++i) {
+        result_data[i] = std::rint(x_ptr[i]);
+    }
+    return MakeArray(chainerx::Dtype::kFloat64, x.shape(), result_data.data()).AsType(x.dtype());
+}
+
+chainerx::Array BitShiftOp::RunImpl(ChxVMState* st, const chainerx::Array& x, const chainerx::Array& in_y) {
+    chainerx::Array int64_y = in_y.BroadcastTo(x.shape()).AsType(chainerx::Dtype::kInt64);
```
Comment (shinh) [inline runtime/ops/quantize.cc]: Better to `CHECK(!IsFloat(x))`?

**Comment by shinh** [comment]
Still looks good to me

**Pull request merged**